delta-proto/src/main/java/io/cdap/delta/proto/Plugin.java

```diff
@@ -44,11 +45,23 @@ public String getType() {
   }
 
   public Map<String, String> getProperties() {
-    return properties;
+    return properties == null ? Collections.emptyMap() : properties;
```

Comment:
Can we have this logic in the constructor instead of get()

Reply:
no because this is normally constructed through gson deserialization. I can add comments so that's clear